tests/vmi_configuration_test.go

```diff
@@ -532,22 +532,24 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			})
 
 			It("[test_id:5266]should boot to NIC rom if a boot order was set on a network interface", func() {
-				By("Creating a VMI with no disk and an explicit network interface")
-				vmi := tests.NewRandomVMI()
-				tests.AddExplicitPodNetworkInterface(vmi)
-
-				By("Enabling BIOS serial output")
-				vmi.Spec.Domain.Firmware = &v1.Firmware{
-					Bootloader: &v1.Bootloader{
-						BIOS: &v1.BIOS{
-							UseSerial: tests.NewBool(true),
-						},
-					},
-				}
-
 				By("Enabling network boot")
 				var bootOrder uint = 1
-				vmi.Spec.Domain.Devices.Interfaces[0].BootOrder = &bootOrder
+				var interfaceDeviceWithMasqueradeBinding = libvmi.InterfaceDeviceWithMasqueradeBinding()
+				interfaceDeviceWithMasqueradeBinding.BootOrder = &bootOrder
+
+				By("Creating a VMI with no disk and an explicit network interface")
+				vmi := libvmi.New(
+					libvmi.WithResourceMemory(func() string {
+						if checks.IsARM64(testsuite.Arch) {
+							return "256Mi"
+						}
+						return "128Mi"
+					}()),
```

Comment:
Personally I don't like this very much.
I would calculate this before:

```go
resourceMemory := "256Mi"
if !checks.IsARM64(testsuite.Arch) {
    resourceMemory = "128Mi"
}

...

vmi := libvmi.New(
    resourceMemory,
    ...
)
```

Reply:
But, thinking about this again, I'm not sure why it's needed at all.
When we convert a VMI into a target pod, we [take care](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/services/template.go#L1479) of such memory allocations.

I think we can remove this altogether, or add this to one of those:
- renderLaunchManifest
- libvmi's New()
- libvmi's constructors (NewFedora, NewCirros, etc).

This can be done in a follow-up PR, but I would at least simply do:
```go
if !checks.IsARM64(testsuite.Arch) {
    resourceMemory = "128Mi"
}
```

Since the other case (where it is an arm64 arch) is taken care of inside renderLaunchManifest

Reply:
Done.